--- master/setup.py.orig	2015-01-12 20:24:22.545577000 +0100
+++ master/setup.py	2015-01-12 20:54:24.944290000 +0100
@@ -215,9 +215,9 @@
     setup_args['install_requires'] += [
         # sqlalchemy-0.8 betas show issues with sqlalchemy-0.7.2, so stick to 0.7.10
         'sqlalchemy >= 0.6, <= 0.7.10',
-        # buildbot depends on sqlalchemy internals, and these are the tested
-        # versions.
-        'sqlalchemy-migrate ==0.6.1, ==0.7.0, ==0.7.1, ==0.7.2',
+        # buildbot depends on sqlalchemy internals, and this is the tested
+        # version.
+        'sqlalchemy-migrate==0.7.2',
         'python-dateutil==1.5',
     ]
     setup_args['tests_require'] = [
